### PR TITLE
KOGITO-3292 KNative eventing is not enabled with Quarkus Extension

### PR DESCRIPTION
**Pull request by evacchi**
https://issues.redhat.com/browse/KOGITO-3292

KNative eventing was erroneously disabled when
KOGITO-3055 Refactor Quarkus Extension (#688)
was merged

### CODE REVIEW

**Review by ricardozanini — APPROVED** [review]
:face_with_thermometer:

**Comment by evacchi** [comment]
> Caused by: java.io.FileNotFoundException: src/test/resources/gateway/EventBasedSplit.bpmn2 (Too many open files)

jenkins please retest this

**Comment by evacchi** [comment]
jenkins please retest this
I don't understand this failure, it works on my machine...

File: kogito-quarkus-extension/deployment/src/main/java/org/kie/kogito/quarkus/deployment/KogitoAssetsProcessor.java [hunk 1]
```diff
@@ -104,6 +104,7 @@
     private static final DotName persistenceFactoryClass = DotName.createSimple("org.kie.kogito.persistence.KogitoProcessInstancesFactory");
     private static final DotName metricsClass = DotName.createSimple("org.kie.kogito.monitoring.rest.MetricsResource");
     private static final DotName tracingClass = DotName.createSimple("org.kie.kogito.tracing.decision.DecisionTracingListener");
+    private static final DotName knativeEventingClass = DotName.createSimple("org.kie.kogito.events.knative.ce.http.HttpRequestConverter");
```
Comment (ricardozanini) [inline kogito-quarkus-extension/deployment/src/main/java/org/kie/kogito/quarkus/deployment/KogitoAssetsProcessor.java:107]: @ruromero I noticed that you removed this class on #750 . I believe we will need to coordinate this change.

File: kogito-codegen/src/main/resources/class-templates/events/CloudEventsMessageProducerTemplate.java [hunk 2]
```diff
@@ -11,14 +11,16 @@
 
 
 import java.util.Optional;
-
+import java.util.TimeZone;
```
Comment (fjtirado) [inline kogito-codegen/src/main/resources/class-templates/events/CloudEventsMessageProducerTemplate.java:14]: Is this import needed?

**Pull request merged**